examples/console/console-security-oidc.yaml

```diff
@@ -107,11 +109,11 @@ spec:
                   - consumerGroups
                   - rebalances
                   - nodes/configs
-              - privileges:
-                  - get
-                  - list
+                privileges:
+                  - GET
+                  - LIST
```

Comment:
It seems there is also a bug when `resourceNames` is not set. I opened https://github.com/streamshub/console/issues/1476